ddtrace/debugging/_probe/remoteconfig.py

```diff
@@ -87,7 +88,7 @@ def build(cls, args: Dict[str, Any], attribs: Dict[str, Any]) -> Any:
         cls.update_args(args, attribs)
 
         where = attribs["where"]
-        if where.get("sourceFile", None) is not None:
+        if where["kind"] == ProbeLocationKind.LINE:
```

Comment:
how does this work with old configs? as the kind field would not be set

Reply:
AFAIK we still don't have support for this in the backend